client/my-sites/plugins/plugin-remove-button/index.jsx

```diff
@@ -88,7 +89,11 @@ class PluginRemoveButton extends Component {
 			);
 		}
 
-		if ( ! this.props.site.canUpdateFiles && this.props.site.options.file_mod_disabled ) {
+		const canRemovePlugin =
+			( this.props.site.canUpdateFiles && this.props.site.options.file_mod_disabled ) ||
```

Comment:
@yashwin This check doesn't work for sites that have only the A4A client plugin installed so I added the check below to check if it's A4A + has plugin capabilities

Reply:
Looks fine. I hope this doesn't affect the non-A4A environments.